CHANGES.rst

```diff
@@ -214,6 +214,9 @@ New Features
 
   - Added ``units.astronomical_unit`` as a long form for ``units.au``. [#3303]
 
+  - Clarified imperial mass measurements and added pound force (lbf),
+    kilopound (kip), and pound per square inch (psi)
```

Comment:
Thanks.  I moved the feature to 1.1 in the changelog and added the [#3409] reference.  This is a simple feature but it will be convenient for those of us who must sometimes convert.